doorway-ui-components/src/icons/Icons.tsx

```diff
@@ -603,6 +603,14 @@ export const Messages = (props: IconProps) => {
   )
 }
 
+export const Minus = (props: IconProps) => {
+  return (
+    <svg viewBox={"0 0 448 512"} fill={props.fill ?? "currentColor"}>
+      <path d="M432 256c0 17.7-14.3 32-32 32L48 288c-17.7 0-32-14.3-32-32s14.3-32 32-32l352 0c17.7 0 32 14.3 32 32z"/>
+    </svg>
+  )
+}
+
```

Comment:
When I download as svg the minus, I get the following values for the path d
`<path d="M6.25 15.9375V14.0625H23.75V15.9375H6.25Z"/>`
and the viewBox is `0 0 30 30`

When this is applied it makes the lines the same as the border (which is @ifranch asks about in a future comment). I suggest we make these changes to Minus / Plus rather than change the border.

And for Plus below the values I see are:
`viewBox={"0 0 30 30"}`
`<path d="M14.0625 23.75V15.9375H6.25V14.0625H14.0625V6.25H15.9375V14.0625H23.75V15.9375H15.9375V23.75H14.0625Z"></path>`

Someone should double check my work tho!

Reply:
Thank you!